README.md

```diff
@@ -15,7 +23,7 @@ it lets you [opt out](#remove-the-aws-luminarlz-cli) and remove the dependency t
 
 This is a step-by-step guide to initialize a new AWS landing zone using the luminarlz CLI.
 
-We'll use the `foundational` blueprint that is heavily aligned with the [Guidance for Establishing an Initial Foundation using Control Tower on AWS](https://aws.amazon.com/solutions/guidance/establishing-an-initial-foundation-using-control-tower-on-aws).
+We'll use the [foundational blueprint](blueprints/foundational/) that is heavily aligned with the [Guidance for Establishing an Initial Foundation using Control Tower on AWS](https://aws.amazon.com/solutions/guidance/establishing-an-initial-foundation-using-control-tower-on-aws).
```

Comment:
I understand this point now. Maybe, since Till and me were looking for the documentation, could there maybe be some sentence like: "You will find more documentation in the initialized `aws-luminarlz-cli` project."? I mean, it might be obvious, but a hint could be nice to catch ppl like me not understanding that the documentation is always project specific.

Reply:
See point 9. ;-)